app/_how-tos/create-custom-dashboards.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Create an Analytics Custom Dashboard from a template
```

Comment:
@Guaris Are we adding more content in here so it matches the "Frank" story or do you plan on having multiple smaller "How tos"?